tests/test_callbacks.cpp

```diff
@@ -117,7 +117,12 @@ TEST_SUBMODULE(callbacks, m) {
         }
     });
 
-    class AbstractBase { public: virtual unsigned int func() = 0; };
+    class AbstractBase {
+    public:
+        AbstractBase(const AbstractBase&) = default;
```

Comment:
Wait, you can't construct and abstract base class. Can you copy-construct it? And why would you need to?

Reply:
Slice a non-abstract "leaf" class and you get an instance of an abstract base class. You just can't construct it directly, but copying definitely works.

However, I also wonder why would a copy constructor be needed.

Reply:
What about `AbstractBase(const AbstractBase&) = delete;`?